designer/server/src/models/forms/notification-email.js

```diff
@@ -0,0 +1,36 @@
+import { buildErrorList } from '~/src/common/helpers/build-error-details.js'
+
+/**
+ * @param {FormMetadata} metadata
+ * @param {ValidationFailure<Pick<FormMetadataInput, 'notificationEmail'>>} [validation]
+ */
+export function notificationEmailViewModel(metadata, validation) {
+  const pageTitle = 'Email address for submitted forms'
+  const { formValues, formErrors } = validation ?? {}
+
+  return {
+    form: metadata,
+    backLink: {
+      text: 'Back',
+      href: `/library/${metadata.slug}`
+    },
```

Comment:
Mind aligning the back links with this last commit https://github.com/DEFRA/forms-designer/pull/463/commits/7b25501340abd0a587bcc3c826cda7ba2bb85f68 ?